wordpress-mobile/WordPress-iOS
# Zendesk Mobile: add support for Version, Extra Debug, Activity Log

Fixes #9049 

**NOTES:**
- This recreates the extra information options in the Support view previously handled by `SupportViewController`.
- Right now, it can be accessed by the 'Help' button on the nav bar. Replacing all instances of `SupportViewController` will come later.
- Support for the help options will come later. Selecting any of those will still display a temporary alert.

**To test:**
- Press 'Help' on the nav bar (ex: from Site Creation or Log In).
- Toggle 'Extra Debug'
  - Relaunch app, go to Support page.
  - Verify Extra Debug is what you set it to.
- Verify 'Version' as the actual version (instead of hardcoded 1.1).
- Verify 'Activity Log' displays activity logs.

@diegoreymendez , @etoledom - can I bother one of you for a quick review please? Thank you!!
